app/screens/edit_post/edit_post.tsx

```diff
@@ -246,7 +294,9 @@ const EditPost = ({componentId, maxPostSize, post, closeButtonId, hasFilesAttach
                                 message={postMessage}
                                 onChangeText={onInputChangeText}
                                 onTextSelectionChange={onTextSelectionChange}
-                                ref={postInputRef}
+                                inputRef={postInputRef}
```

Comment:
`EditPostInput` component is not wrapped with HOC form WatermelonDB, but this HOC don't automatically forward refs when trying to use `React.forwardRef`, which was causing ref forwarding issues. 

To resolve this issue, instead of forwarding Ref, directly pass it as a prop.